content/en/community/hackathons/sessions/complex-applications/content/Work-Items_05.-Redis-Benchmarking-(Tutorial).md

```diff
@@ -27,22 +27,22 @@ Usage: redis-benchmark [-h <host>] [-p <port>] [-c <clients>] [-n <requests>] [-
 
 If everything runs as expected, you'll see the following output:
 
-```
-"SET","147058.81"
-"GET","153846.16"
+```bash
+"SET","85470.09"
+"GET","83333.34"
```

Comment:
These values are pretty low and make unikraft look bad (compared to native linux :sweat_smile: )

Make we could try changing/running the example with initrd? 9pfs slows things down quite a bit